app/lib/katello/validators/content_view_environment_validator.rb

```diff
@@ -1,5 +1,6 @@
 module Katello
   module Validators
+    # used by activation key and content facet
```

Comment:
I first said to myself, Why isn't it already hitting this? but then I realized this validator is only used by ActivationKey and ContentFacet records, not ContentViewEnvironment itself. So I made a separate one for this case.